packages/editor/src/store/effects/posts.js

```diff
@@ -312,7 +312,8 @@ export const refreshPost = async ( action, store ) => {
 	const postTypeSlug = getCurrentPostType( getState() );
 	const postType = await resolveSelector( 'core', 'getPostType', postTypeSlug );
 	const newPost = await apiFetch( {
-		path: `/wp/v2/${ postType.rest_base }/${ post.id }?context=edit`,
+		// Use timestamp to bypass caching
```

Comment:
Should this "bust cache" behavior be a feature of `apiFetch()` instead of being handled by the caller?

Reply:
I suggest this change because the preloading middleware is not part of `apiFetch` which means users of `apiFetch` shouldn't be aware of its existence. Having a `timestamp` seems like a middle ground solution between being aware of a caching mechanism and explicitely stating that this call being time dependent.

Reply:
Ok. This rationale makes sense to me. We should update the code comment to explain _why_ instead of what (which is obvious by the code).

Reply:
Maybe something like:
> // Allows caller to bypass browser caching, which is expected for this specific function.

Reply:
I agree with @danielbachhuber that in the long run, it should be possible to invalidate the cache on demand in the preloading middleware.

Reply:
For me it almost feels like the preloading middleware should invalidate itself on first call.

Reply:
Hmm, if you assume that all fetched data is stored in data stores then we could make it work it this way.

Reply:
> For me it almost feels like the preloading middleware should invalidate itself on first call.

I can try and work on that, but I'm a bit short on time since we're trying to get this fixed in time for Jetpack's code freeze tomorrow, and the self-invalidation seems to have more impact, and cases that need testing. Would it be okay to go with this PR for now and implement the self-invalidation later?

Reply:
yes, let's get it in.